src/main/java/com/exadel/sandbox/config/Config.java

```diff
@@ -0,0 +1,51 @@
+package com.exadel.sandbox.config;
+
+import com.exadel.sandbox.mapper.AttachmentMapper;
+import com.exadel.sandbox.mapper.FloorMapper;
+import com.exadel.sandbox.mapper.OfficeMapper;
+import com.exadel.sandbox.mapper.impl.AttachmentMapperImpl;
+import com.exadel.sandbox.mapper.impl.FloorMapperImpl;
+import com.exadel.sandbox.mapper.impl.OfficeMapperImpl;
+import com.exadel.sandbox.service.AttachmentService;
+import com.exadel.sandbox.service.FloorService;
+import com.exadel.sandbox.service.OfficeService;
+import com.exadel.sandbox.service.impl.AttachmentServiceImpl;
+import com.exadel.sandbox.service.impl.FloorServiceImpl;
+import com.exadel.sandbox.service.impl.OfficeServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class Config {
+    @Bean
+    AttachmentService attachmentService() {
+        return new AttachmentServiceImpl();
+    }
+
+    @Bean
```

Comment:
We don't need to create BEAN for services. instead of this you can use @conponent or @service annotations.